integrations/azure/README.md

```diff
@@ -0,0 +1,47 @@
+# Azure
```

Comment:
Talk to sinai
If im not wrong it can be deployed using azure
So it shouldnt show the helm installation